docs/Contributing/Blueprints/Fleet-Installers.md

```diff
@@ -0,0 +1,431 @@
+| fleet-certificate | string  | body | Server certificate chain                                                               |
+| insecure          | boolean | body | Disable TLS certificate verification (default: false)                                  |
+| osqueryd-channel  | string  | body | Update channel of osqueryd to use (default: "stable")                                  |
+| desktop-channel   | string  | body | Update channel of desktop to use (default: "stable")                                   |
+| orbit-channel     | string  | body | Update channel of Orbit to use (default: "stable")                                     |
+| disable-updates   | boolean | body | Disable auto updates on the generated package (default: false)                         |
+| debug             | boolean | body | Enable debug logging in orbit (default: false)                                         |
+| fleet-desktop     | boolean | body | Include the Fleet Desktop Application in the package (default: false)                  |
+| update-interval   | string  | body | Interval that Orbit will use to check for new updates (10s, 1h, etc.) (default: 15m0s) |
+| osquery-flagfile  | string  | body | Flagfile to package and provide to osquery (default: empty)                            |
+| service           | boolean | body | Install with a persistence service (launchd, systemd, etc.) (default: true)            |
+
+##### Response Fields
+
+| Name              | Type    | In   | Description                      |
+| ----------------- | ------- | ---- | -------------------------------- |
+| package_id        | string  | body | ID of the package being created. |
+
+#### 2. Package Status Check
+
+`GET /status`
+
+This endpoint allows checking the status of a package being created.
+Clients can poll for the status of a package using this API.
+
+This endpoint should be rate-limited by `package_id`.
+
+##### Request Fields
+
+| Name              | Type    | In    | Description                                                              |
+| ----------------- | ------- | ----- | ------------------------------------------------------------------------ |
+| package_id        | string  | query | **Required.** ID of the package created with `POST /create` |
+
+##### Response Fields
+
+| Name              | Type    | In   | Description                                                              |
+| ----------------- | ------- | ---- | ------------------------------------------------------------------------ |
+| status            | string  | body | One of the following values "success", "fail", "in-progress", "expired"  |
+| download_url      | string  | body | Set to the download URL if status field is "success"                     |
+| progress          | string  | body | Set to the progress (0-99) if status field is "in-progress"              |
```

Comment:
Indeed. It would not be accurate. Am ok with not returning this.
Alternatively, progress should be replaced by "stage", which returns the stage of the package generation, e.g. "downloading targets", "signing", "notarizing", etc.